paramiko/auth_handler.py

```diff
@@ -566,7 +604,6 @@ def _parse_userauth_failure(self, m):
     def _parse_userauth_banner(self, m):
         banner = m.get_string()
         self.banner = banner
-        lang = m.get_string()
```

Comment:
just pointing out another not-just-whitespace change (which is probably OK)
(but you can't always drop `m.get_string()` when the result is unused, because it mutates `m`)